### DISCUSSION TITLE
[Feature Enhancement & Request] Layout Enhancement & Request on different views

### DISCUSSION THREAD

**Discussion by vnana-git** [marcusolsson/obsidian-projects#508] [General]
### What would you like to be added?

Hello, really love your creating concept(watched the interview in YouTube) and thank you for making this plugin! Love watching this plugin getting better and better on every updates!!!

Would love to have below layout enhancements:

**🎨 Gallery View**  
 1. Option to hide image in the gallery view if image metadata is empty, and if possible I wish we can have a wall view display on that (no fix height)
2. Option to hide the default page name, as wanna to use `Include fields > name` instead
3. Option to hide field names on the gallery card, by field name which mean the title of metadata, e.g. tags, aliases
4. Display tag like tags in pages

**📆 Calendar View**
1. Option to display tags in calendar
2. Option to hide the default page name, instead we can select to show `Include fields > name` instead

**On all views**
- #439 
Hover to pop the page content for quick page edit. (personally using the Hover Editor plugin for that)
- Display tag like tags in pages on all the views

### Why is this needed?

**🎨 Gallery View**  
1. Wish to seamlessly browse through all the content as wanna to create my own love wall using the Project plugin
2. My vault is using Supercharged Links to add emojis and colour on my pages. That helps me visually identify different types and status on all my pages. So personally would want that to be visually view on gallery or calendar
3. Just an enhancement wish, as normally not all field names needs to be shown
4. Just an enhancement wish also

**📆 Calendar View**
1. An enhancement wish, easier for page management
2. Same reason as 🎨 Gallery View No.2

**On all views**
- Those are all enhancement wishes, easier for page edit and management, functionally and visually

Thank you!!

**Comment by marcusolsson** [comment]
Wow! Thank you for all the great suggestions! 🙏 I think all of them are both reasonable and doable.

I'd like to ask that you create separate issues for each feature request. It would really help when prioritizing and discussing each feature request. Instead of closing this issue, I moved it into Discussions.
